tests/testthat/test-toolbar.R

```diff
@@ -0,0 +1,55 @@
+test_that("toolbar() basic attributes and defaults", {
+    tb <- as.tags(toolbar(htmltools::span("Test")))
```

Comment:
Interesting that this is indented by 4 spaces. Check out https://github.com/posit-dev/air, its auto-formatting is super fast and keeps us all in-sync. There's a VS Code extension, too